projects/packages/my-jetpack/_inc/components/plans-section/index.jsx

```diff
@@ -165,18 +162,22 @@ function PlanSectionFooter( { purchases } ) {
  */
 export default function PlansSection() {
 	const userIsAdmin = !! window?.myJetpackInitialState?.userIsAdmin;
-	const { purchases } = usePurchases();
+	const { data: purchases, isLoading, isError } = usePurchases();
+
+	const isDataLoaded = purchases && ! isLoading && ! isError;
+	const numberOfPurchases = isDataLoaded ? purchases.length : 0;
 
 	return (
 		<div className={ styles.container }>
-			<PlanSectionHeader purchases={ purchases } />
+			<PlanSectionHeader purchases={ numberOfPurchases } />
```

Comment:
Here, the prop name changed
```suggestion
			<PlanSectionHeader numberOfPurchases={ numberOfPurchases } />
```

Reply:
Oh nice catch, thank you